src/mono/wasm/runtime/binding_support.js

```diff
@@ -123,6 +123,8 @@ var BindingSupportLib = {
 			this.safehandle_get_handle = get_method ("SafeHandleGetHandle");
 			this.safehandle_release_by_handle = get_method ("SafeHandleReleaseByHandle");
 
+			this.object_to_enum = get_method ("ObjectToEnum");
```

Comment:
Just trying to understand - I see `object_to_enum` being used in `js_to_mono_enum`, but it is being added here. How was it working earlier?

Reply:
It was not working before.  Thus the issue and pr